db/migrate/20220827155600_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
```

Comment:
Why do we need commenter field, if commenter it's a user, who added comment? I think, we can remove this field.

Reply:
Any updates here?

Reply:
I made a migration to remove the column. Should I remove the line directly in the migration?

Reply:
Oh, it's fine. I have. not seen it.